addons/mrp/models/mrp_bom.py

```diff
@@ -114,6 +114,14 @@ def name_create(self, name):
     def name_get(self):
         return [(bom.id, '%s%s' % (bom.code and '%s: ' % bom.code or '', bom.product_tmpl_id.display_name)) for bom in self]
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        product_tmpl_ids = [vals['product_tmpl_id'] for vals in vals_list if vals.get('type') == 'phantom']
+        product_ids = self.env['product.template'].browse(product_tmpl_ids).mapped('product_variant_ids').ids
```

Comment:
I suggest you to make the search after the call to super because you can create BoM for a specific variant. In that case, only this product is an issue regarding the orderpoint not the other variant of the template